webapplication/aoi/models.py

```diff
@@ -119,10 +119,16 @@ class Request(models.Model):
     calculated = models.BooleanField(default=False, verbose_name='Notebook calculated')
     success = models.BooleanField(default=False, verbose_name='Execution succeeded')
     error = models.CharField(max_length=400, blank=True, null=True, verbose_name='Error')
+    user_error = models.CharField(max_length=400, blank=True, null=True, verbose_name='User Error')
     polygon = models.PolygonField(spatial_index=True, verbose_name='Polygon')
     additional_parameter = models.CharField(max_length=50, null=True, blank=True, verbose_name='Additional parameter')
     request_origin = models.CharField(max_length=50, default="https://portal.soilmate.ai/", verbose_name='Request origin')
 
     @property
     def component_name(self):
         return self.component.name
+
+
+class UserError(models.Model):
+    user_error = models.CharField(max_length=400, blank=True, null=True, verbose_name='User error')
+    component_error = models.CharField(max_length=400, blank=True, null=True, unique=True, verbose_name='Component error')
```

Comment:
The idea is good, although naming is not ok. Looking at the model I do not understand what model and it attiubutes represent.